kubeflow/instance/gcp_config/cluster-patch.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is a strategic patch on top of Kubeflow default cluster specification.
+# You can freely edit this to customize your cluster.
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: KUBEFLOW-NAME # {"$kpt-set":"name"}
+# To replace default nodepool with your own, uncomment the following two lines
+# and add "nodepool-example.yaml" to "resources" field of kustomization.yaml.
+# annotations:
+#   cnrm.cloud.google.com/remove-default-node-pool: "false"
+spec:
+  location: "LOCATION" # {"$kpt-set":"location"}
```

Comment:
nit: quoting or not, let's keep it consistent for `KUBEFLOW-NAME` and `LOCATION`

(IIUC, this the the source, and value flows into `Kptfile`, right?)